resolwe_bio/processes/alignment/star.yml

```diff
@@ -144,6 +150,58 @@
           required: false
           description: >
             Maximum gap between two mates, if 0, max intron gap will be determined by (2pow(winBinNbits)*winAnchorDistNbins) (default: 0).
+        - name: alignEndsType
+          label: --alignEndsType
+          type: basic:string
+          required: false
+          description: >
+            Type of read ends alignment (default: Local).
+          default: Local
+          choices:
+            - label: Local
+              value: Local
+            - label: EndToEnd
+              value: EndToEnd
+            - label: Extend5pOfRead1
+              value: Extend5pOfRead1
+            - label: Extend5pOfReads12
+              value: Extend5pOfReads12
+    - name: output_sam_bam
+      label: Output SAM and BAM
+      group:
+        - name: outSAMunmapped
+          label: --outSAMunmapped
+          type: basic:string
+          description: >
+            Output of unmapped reads in the SAM format.
+          required: false
+          default: None
+          choices:
+            - label: None
+              value: None
+            - label: Within
+              value: Within
+        - name: outSAMattributes
```

Comment:
If `outSAMunmapped` is left to `None`, will the use of other related parameters make the process to fail?

Reply:
If `--outSAMunmapped` is set to None, there just won't be any unaligned reads in file `Aligned.sortedByCoord.out.bam`. I don't see how this could cause failure.